pyproject.toml

```diff
@@ -68,21 +70,21 @@ module = [
     "bokeh.driving",
     "bokeh.embed.standalone",
     "bokeh.embed.wrappers",
-    "bokeh.ext",
-    "bokeh.io",
+    "bokeh.ext.*",
+    "bokeh.io", # TODO: .*;  mypy_path resolution is broken
     "bokeh.palettes",
     "bokeh.plotting._tools",
-    "bokeh.protocol",
-    "bokeh.sampledata",
+    "bokeh.protocol.*",
```

Comment:
Please do conduct a test to make sure that `bokeh.protocol` is not needed in addition to `bokeh.protocol.*` etc. My recollection is that the latter only covers submodules an not the package itself

Reply:
I tested and `.*` covers everything.